File: default_output/flutter/connect_notification/connect_notification_migration.dart
class Database
{
    void execute(String s) {}
}

class ConnectNotificationMigration
{
    void createDatabase(Database db)
    {
        db.execute("CREATE TABLE ConnectNotification"
              +"connect_notification_uuid TEXT, "
              +"public_key TEXT, "
              +"notified_user_uuid TEXT, "
              +"username TEXT, "
              +"vorname TEXT, "
              +"nachname TEXT, "
              +"photo TEXT, "
              +"thumbnail TEXT, "
              +"description TEXT, "
              +"user_id INTEGER, "
              +"user_uuid TEXT"
            ")"
        );
    }
}

File: default_output/phoenix/connect_notification/connect_notification_migration.ex
defmodule FinalLocallity.Repo.Migrations.CreateConnectNotification do
  use Ecto.Migration

  def change do
    create table(:connect_notification) do
      add :connect_notification_uuid, :string
      add :public_key, :string
      add :notified_user_uuid, :string
      add :username, :string
      add :vorname, :string
      add :nachname, :string
      add :photo, :string
      add :thumbnail, :string
      add :description, :string
      add :user_id, :integer
      add :user_uuid, :string

      timestamps()
    end

  end
end
